compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.h

```diff
@@ -104,15 +104,41 @@ FailureOr<Flow::DispatchRegionOp> wrapOpInDispatchRegion(RewriterBase &rewriter,
 /// into a dispatch region.
 bool isClonableIntoDispatchOp(Operation *op);
 
-/// Returns true if the operation has dequantization-like properties.
+/// Returns true if the operation increases/decreases bitwidths of tensors.
 /// This function checks that the genericOp:
```

Comment:
So this would exclude future cast ops that convert between types of the same bitwidth (say two different fp8 types)?

Reply:
I think cast of same bitwidth shouldnt be accounted for here. This is about getting the result written out in the smaller bitwidth.